Regarding New JIRA plugins 1.0.20 and Update doc

SagarQA1695 opened the pull request:
Dear Team,
Release a new build version 1.0.20 with fixes the issues listed below and also update the admin guide document's. 1.JIRA SAML SSO add-on redirects to incorrect URL from mobile browser. 2.The mark log section after enabling the JIRA plugin. 3.JIRA plugin needs to report the last login date in the JIRA log. The issue is in Jira itself, the Last Login Date for a user does not update when sign in is via SSO

SagarQA1695 committed:
Regarding New JIRA plugins 1.0.20 and Update doc

Dear Team,
Release a new build version 1.0.20 with fixes the issues listed below and also update the admin guide document's.
1.JIRA SAML SSO add-on redirects to incorrect URL from mobile browser.
2.The mark log section after enabling the JIRA plugin.
3.JIRA plugin needs to report the last login date in the JIRA log. The issue is in Jira itself, the Last Login Date for a user does not update when sign in is via SSO

SagarQA1695 committed:
Update ms-confluence-jira-plugin-adminguide.md

Modified Release Note table format

SagarQA1695 reviewed (commented):
Hello @dhivyagana,
I've changed the Release Notes Table Format, and I'm requesting you to confirm it on your end.

Thnx in advance.

SagarQA1695 committed:
Update ms-confluence-jira-plugin-adminguide.md

New Modification.

SagarQA1695 reviewed (commented):
Hello @dhivyagana,
I've changed the Release Notes Table Format, and also review from my end now changes reflect as intended.
Requesting you to confirm the same from your end and let me know incase of any.

SagarQA1695 reviewed (commented):
Hello @dhivyagana 
I've changed the Release Notes Table Format, and also review from my end now changes reflect as intended.
Requesting you to confirm the same from your end and let me know incase of any.

dhivyagana commented on articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md:
```diff
@@ -123,7 +123,29 @@ The following image shows the configuration screen in both Jira and Confluence:
+|  1.0.20         |   Bug Fixes:                                                                              | Jira Core and Software:             |
+|                 |   1. JIRA SAML SSO add-on redirects to incorrect URL from mobile browser.                 |  7.0.0 to 9.5.0                     |
+|                 |   2. The mark log section after enabling the JIRA plugin.                                 |                                     |
+|                 |   3. JIRA plugin needs to report the last login date in the JIRA log.The issue is in Jira itself, the Last Login Date for a user does not update when sign in is via SSO.                                                                                      |                                     |
```
minor sentence correction : the last login date for a user does not update **when user signs in via SSO**

SagarQA1695 committed:
Update ms-confluence-jira-plugin-adminguide.md

Hello Dhivya,
I have restructured the sentence and verified on my end, please check and let me know if any.

SagarQA1695 reviewed (commented):
@dhivyagana, I have restructured the sentence and verified on my end, please check and confirm the same.

dhivyagana reviewed (approved):
#sign-off

Court72 commented on articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md:
```diff
@@ -123,7 +123,29 @@ The following image shows the configuration screen in both Jira and Confluence:
+|                 |    2. JIRA server is not rendering the "Project Setting Page" correctly.                  | Jira Service Desk: 3.0.0 to 4.22.1. |
+|                 |    3. JIRA is not forcing Azure AD Login, extra button click was required.                |                                     |
+|                 |    4. We have now resolved the security fix in this version.This will protect you from user impersonation vulnerability.|       |
+|                 |    5. JIRA Service Desk logout issue is resolved.                                         |                                     |
```
```suggestion
|                 |    Bug fix for the 405 error upon clicking on the Configure button of the Jira Azure AD SSO Plugin.|  6.0 to 9.1.0.          |
|                 |    JIRA server isn't rendering the "Project Setting Page" correctly.                  | Jira Service Desk: 3.0.0 to 4.22.1. |
|                 |    JIRA isn't forcing Azure AD Login. An extra button click was required.                |                                     |
|                 |    We have now resolved the security fix in this version. This will protect you from user impersonation vulnerability.|       |
|                 |    JIRA Service Desk logout issue is resolved.                                         |                                     |
```

Court72 commented on articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md:
```diff
@@ -123,7 +123,29 @@ The following image shows the configuration screen in both Jira and Confluence:
+|  1.0.20         |   Bug Fixes:                                                                              | Jira Core and Software:             |
+|                 |   1. JIRA SAML SSO add-on redirects to incorrect URL from mobile browser.                 |  7.0.0 to 9.5.0                     |
+|                 |   2. The mark log section after enabling the JIRA plugin.                                 |                                     |
+|                 |   3. The last login date for a user does not update when user signs in via SSO            |                                     |
```
```suggestion
|                 |   JIRA SAML SSO add-on redirects to incorrect URL from mobile browser.                 |  7.0.0 to 9.5.0                     |
|                 |   The mark log section after enabling the JIRA plugin.                                 |                                     |
|                 |   The last login date for a user doesn't update when user signs in via SSO            |                                     |
```

Court72 commented on articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md:
```diff
@@ -123,7 +123,29 @@ The following image shows the configuration screen in both Jira and Confluence:
 
    * For App proxy setup , follow the steps on the [Azure AD App Proxy Documentation](../app-proxy/what-is-application-proxy.md).
-
+   * 
```
```suggestion

```

Court72 committed:
Apply suggestions from PR review
